hatchery/gen3licenseusermaps.go

```diff
@@ -99,11 +99,16 @@ func getItemsFromQuery(dbconfig *DbConfig, queryInput *dynamodb.QueryInput) ([]m
 	return tableItems, nil
 }
 
-var getActiveGen3LicenseUserMaps = func(dbconfig *DbConfig, container Container) (gen3LicenseUserMaps *[]Gen3LicenseUserMap, err error) {
+var getActiveGen3LicenseUserMaps = func(dbconfig *DbConfig, container Container) (gen3LicenseUserMaps []Gen3LicenseUserMap, err error) {
 	// Query the table to get all active gen3 license user map items
+	emptyList := []Gen3LicenseUserMap{}
 
 	targetEnvironment := os.Getenv("GEN3_ENDPOINT")
-	validateContainerLicenseInfo(container.Name, container.License)
+	ok := validateContainerLicenseInfo(container.Name, container.License)
+	if !ok {
```

Comment:
by convention, the way to do this in Go is with errors

this function should return an error instead of a boolean https://github.com/uc-cdis/hatchery/blob/b32a3bc/hatchery/gen3licenseusermaps.go#L49
and the calling functions can choose what to do with the error (throw it or not - here, ignore it and return an empty list)

and side question - should the returned value by checked here too? https://github.com/uc-cdis/hatchery/blob/b32a3bc/hatchery/gen3licenseusermaps.go#L301
(if you use an error instead of a bool, i think Go refuses to build if you don't check the returned error, which helps not to miss something like this)

Reply:
Nice! Return type for `validateContainerLicenseInfo` has been updated to `error`.